exonum-java-binding/doc/Migration_guide_0.9.md

```diff
@@ -0,0 +1,179 @@
+# Migrating Java Services to 0.9
+
+The flagship feature of Exonum Java 0.9 release is support for dynamic services&nbsp;—
+services that can be added to the blockchain network after it has been started;
+and multiple instances of the same service.
+
+This guide provides instructions on updating existing services to be compatible with the new
+and updated features.
+
+## Dynamic Services 101
```

Comment:
maybe I'm out of the context but what is 101?

Reply:
https://en.wikipedia.org/wiki/101_(topic)

Reply:
It is a number usually assigned to introductory courses, but that is US-specific. Could change to 'Overview'